### PR TITLE
CC on Resources for OSPOs

### LINKED ISSUE #1582: Resources for Open Source Program Offices
https://github.com/github/github-ospo

### PR DESCRIPTION
**Pull request by pagrubel**
# Description
Resources for Open Source Program Offices and Projects

Author: @pagrubel 
EB Member: @`<eb-member-id>`

Resolves #1582 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [ ] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [ ] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [ ] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

### CODE REVIEW

**Comment by pagrubel** [comment]
@davidbernholdt Would you know why the check for external contributions is failing. I'm a team member.

**Comment by pagrubel** [comment]
@markcmiller86 I think this is ready for your review.

File: CuratedContent/ResourcesforOSPO.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#### Publication date: March 28, 2023
+
+Resource information | Details
+:--- | :---
+Organization type  | Open Source Program Offices
+Websites  |  Github OSPO  - [Github site to enable success on github for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
+Focus | Success for Open Source Program Offices (OSPOs)
+
+In some organizations the governance and guidance for open source projects fall under the guize of an Open Source Program Office.
```
Comment (markcmiller86) [inline CuratedContent/ResourcesforOSPO.md:17]:
```suggestion
In some organizations the governance and guidance for open source projects falls under the purview of an Open Source Program Office.
```

File: CuratedContent/ResourcesforOSPO.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+The community is beginning to stand up resources to enable sharing knowledge and guidance for the creation and success of an OSPO.
+The resources outlined in this article will help an organization to create an OSPO and to aid in the success of existing OSPOs.
+The resources are also useful for anyone who manages, maintains and or guides open source projects.
+There are even resources for open source developers, especially in the [Open Source Guides](https://opensource.guide/).
+
+Github is one of the main sites for hosting open source projects.
+The purpose of [Github OSPO](https://github.com/github/github-ospo), a github site itself, is to help OSPOs be successful on github.
```
Comment (markcmiller86) [inline CuratedContent/ResourcesforOSPO.md:24]:
```suggestion
The purpose of [Github OSPO](https://github.com/github/github-ospo), a GitHub site itself, is to help OSPOs be successful on GitHub.
```

I wasn't sure why GitHub was capitalized sometimes and not others. I may be missing something about its usage here though.

File: CuratedContent/ResourcesforOSPO.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+The site includes information about policies for releasing and publishing open source software, guides for repositories and much more.
+
+[OSPO Alliance](https://ospo-alliance.org/) is an open community aimed at sharing knowledge and experience throughout the community. The alliance develops and maintains the OSS Good Governance Initiative (GGI) and hosts the GGI Handbook for setting up and managing OCPOs. The site includes upcoming OSPO OnRamp meetings and links to previously recorded meetings.
+
+The [Open Source Guides](https://opensource.guide/) are numerous guides useful for anyone involved in open source projects.
```
Comment (markcmiller86) [inline CuratedContent/ResourcesforOSPO.md:29]:
```suggestion
The [Open Source Guides](https://opensource.guide/) are various guides which will be helpful to anyone involved in open source projects.
```

File: CuratedContent/ResourcesforOSPO.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Resource information | Details
+:--- | :---
+Organization type  | Open Source Program Offices
+Websites  |  Github OSPO  - [Github site to enable success on github for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
+Focus | Success for Open Source Program Offices (OSPOs)
+
+In some organizations the governance and guidance for open source projects fall under the guize of an Open Source Program Office.
+The community is beginning to stand up resources to enable sharing knowledge and guidance for the creation and success of an OSPO.
+The resources outlined in this article will help an organization to create an OSPO and to aid in the success of existing OSPOs.
```
Comment (markcmiller86) [inline CuratedContent/ResourcesforOSPO.md:19]:
```suggestion
The resources outlined in this article will help organizations to create an OSPO and to aid in the success of existing OSPOs.
```

File: CuratedContent/ResourcesforOSPO.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+<!--deck text end-->
+
+#### Contributed by [Patricia Grubel](http://github.com/pagrubel)
+
+#### Publication date: March 28, 2023
+
+Resource information | Details
+:--- | :---
+Organization type  | Open Source Program Offices
+Websites  |  Github OSPO  - [Github site to enable success on github for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
```
Comment (markcmiller86) [inline CuratedContent/ResourcesforOSPO.md:14]:
```suggestion
Websites  |  Github OSPO  - [Github site to enable success on GitHub for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
```

**Review by markcmiller86 — COMMENTED** [review]
I've proposed some minor edits. Please have a look.

File: CuratedContent/ResourcesforOSPO.md (hunk 1 above)
Comment (pagrubel) [reply CuratedContent/ResourcesforOSPO.md:17]: governance and guidance fall - so not accepting this change

**Comment by pagrubel** [comment]
@markcmiller86 I made the changes except where I noted with a comment. I'm trying to check in on preview but that isn't working right now.

**Comment by pagrubel** [comment]
@markcmiller86 I've checked this on the preview site and am ready to merge it. Would you give it one more look, thanks!

**Review by markcmiller86 — APPROVED** [review]
LGTM

**Pull request merged**